diff -urN old/old-configure.in new/old-configure.in
--- old/old-configure.in	2018-05-28 07:31:25.762391253 -0000
+++ new/old-configure.in	2018-05-28 07:37:50.042398364 -0000
@@ -1261,37 +1261,37 @@
 dnl Ordering is important.
 dnl Xt is dependent upon SM as of X11R6
 dnl ========================================================
-if test -n "$MOZ_X11"; then
-    AC_DEFINE_UNQUOTED(FUNCPROTO,15)
-    _SAVE_LDFLAGS="$LDFLAGS"
-    _SAVE_LIBS="$LIBS"
-    LDFLAGS="$XLDFLAGS $LDFLAGS"
-    AC_CHECK_LIB(X11, XDrawLines, [XLIBS="-lX11 $XLIBS"],
-        [MISSING_X="$MISSING_X -lX11"], $XLIBS)
-    AC_CHECK_LIB(Xext, XextAddDisplay, [XEXT_LIBS="-lXext"],
-        [MISSING_X="$MISSING_X -lXext"], $XLIBS)
-
-    AC_CHECK_LIB(Xt, XtFree, [ XT_LIBS="-lXt"], [
-        unset ac_cv_lib_Xt_XtFree
-        AC_CHECK_LIB(ICE, IceFlush, [XT_LIBS="-lICE $XT_LIBS"],, $XT_LIBS $XLIBS)
-        AC_CHECK_LIB(SM, SmcCloseConnection, [XT_LIBS="-lSM $XT_LIBS"],, $XT_LIBS $XLIBS)
-        AC_CHECK_LIB(Xt, XtFree, [ XT_LIBS="-lXt $XT_LIBS"],
-            [MISSING_X="$MISSING_X -lXt"], $X_PRE_LIBS $XT_LIBS $XLIBS)
-        ])
+dnl if test -n "$MOZ_X11"; then
+dnl    AC_DEFINE_UNQUOTED(FUNCPROTO,15)
+dnl    _SAVE_LDFLAGS="$LDFLAGS"
+dnl    _SAVE_LIBS="$LIBS"
+dnl    LDFLAGS="$XLDFLAGS $LDFLAGS"
+dnl    AC_CHECK_LIB(X11, XDrawLines, [XLIBS="-lX11 $XLIBS"],
+dnl        [MISSING_X="$MISSING_X -lX11"], $XLIBS)
+dnl    AC_CHECK_LIB(Xext, XextAddDisplay, [XEXT_LIBS="-lXext"],
+dnl        [MISSING_X="$MISSING_X -lXext"], $XLIBS)
+
+dnl    AC_CHECK_LIB(Xt, XtFree, [ XT_LIBS="-lXt"], [
+dnl        unset ac_cv_lib_Xt_XtFree
+dnl        AC_CHECK_LIB(ICE, IceFlush, [XT_LIBS="-lICE $XT_LIBS"],, $XT_LIBS $XLIBS)
+dnl        AC_CHECK_LIB(SM, SmcCloseConnection, [XT_LIBS="-lSM $XT_LIBS"],, $XT_LIBS $XLIBS)
+dnl        AC_CHECK_LIB(Xt, XtFree, [ XT_LIBS="-lXt $XT_LIBS"],
+dnl            [MISSING_X="$MISSING_X -lXt"], $X_PRE_LIBS $XT_LIBS $XLIBS)
+dnl        ])
 
     dnl ========================================================
     dnl = Check for xcb
     dnl ========================================================
-    AC_CHECK_LIB(xcb, xcb_connect, [XLIBS="-lxcb $XLIBS"],
-        [MISSING_X="$MISSING_X -lxcb"], $XLIBS)
-    AC_CHECK_LIB(xcb-shm, xcb_shm_query_version, [XLIBS="-lxcb-shm $XLIBS"],
-        [MISSING_X="$MISSING_X -lxcb-shm"], $XLIBS)
-    AC_CHECK_LIB(X11-xcb, XGetXCBConnection, [XLIBS="-lX11-xcb $XLIBS"],
-        [MISSING_X="$MISSING_X -lX11-xcb"], $XLIBS)
-
-    LDFLAGS="$_SAVE_LDFLAGS"
-    LIBS="$_SAVE_LIBS"
-fi # $MOZ_X11
+dnl    AC_CHECK_LIB(xcb, xcb_connect, [XLIBS="-lxcb $XLIBS"],
+dnl        [MISSING_X="$MISSING_X -lxcb"], $XLIBS)
+dnl    AC_CHECK_LIB(xcb-shm, xcb_shm_query_version, [XLIBS="-lxcb-shm $XLIBS"],
+dnl        [MISSING_X="$MISSING_X -lxcb-shm"], $XLIBS)
+dnl    AC_CHECK_LIB(X11-xcb, XGetXCBConnection, [XLIBS="-lX11-xcb $XLIBS"],
+dnl        [MISSING_X="$MISSING_X -lX11-xcb"], $XLIBS)
+
+dnl    LDFLAGS="$_SAVE_LDFLAGS"
+dnl    LIBS="$_SAVE_LIBS"
+dnl fi # $MOZ_X11
 
 AC_SUBST_LIST(XCFLAGS)
 AC_SUBST_LIST(XLDFLAGS)
@@ -2126,9 +2126,9 @@
     GLIB_VERSION_MAX_ALLOWED=$GLIB_VERSION_MIN_REQUIRED
   fi
   if test "$MOZ_ENABLE_GTK"; then
-    if test "$MOZ_X11"; then
-      GDK_PACKAGES=gdk-x11-2.0
-    fi
+dnl    if test "$MOZ_X11"; then
+dnl      GDK_PACKAGES=gdk-x11-2.0
+dnl    fi
     AC_DEFINE_UNQUOTED(GLIB_VERSION_MIN_REQUIRED,$GLIB_VERSION_MIN_REQUIRED)
     AC_DEFINE_UNQUOTED(GLIB_VERSION_MAX_ALLOWED,$GLIB_VERSION_MAX_ALLOWED)
 
@@ -2238,42 +2238,42 @@
 dnl = GConf support module
 dnl ========================================================
 
-if test "$MOZ_X11"
-then
-    if test "$MOZ_ENABLE_GTK"
-    then
-        MOZ_ENABLE_GCONF=1
-    fi
+dnl if test "$MOZ_X11"
+dnl then
+dnl    if test "$MOZ_ENABLE_GTK"
+dnl    then
+dnl        MOZ_ENABLE_GCONF=1
+dnl    fi
 
     dnl ========================================================
     dnl = GConf support module
     dnl ========================================================
-    MOZ_ARG_DISABLE_BOOL(gconf,
-    [  --disable-gconf      Disable Gconf support ],
-        MOZ_ENABLE_GCONF=,
-        MOZ_ENABLE_GCONF=1)
+dnl    MOZ_ARG_DISABLE_BOOL(gconf,
+dnl    [  --disable-gconf      Disable Gconf support ],
+dnl        MOZ_ENABLE_GCONF=,
+dnl        MOZ_ENABLE_GCONF=1)
+
+dnl    if test "$MOZ_ENABLE_GCONF"
+dnl    then
+dnl        PKG_CHECK_MODULES(MOZ_GCONF, gconf-2.0 >= $GCONF_VERSION gobject-2.0 ,[
+dnl            MOZ_GCONF_LIBS=`$PKG_CONFIG --libs gobject-2.0`
+dnl            MOZ_ENABLE_GCONF=1
+dnl        ],[
+dnl            if test -n "$MOZ_ENABLE_GCONF";
+dnl            then
+dnl                AC_MSG_ERROR([* * * Could not find gconf-2.0 ])
+dnl            else
+dnl                AC_MSG_WARN([Many automated tests will fail with --disable-gconf. See bug 1167201.])
+dnl            fi
+dnl        ])
+dnl    fi
+
+dnl    if test "$MOZ_ENABLE_GCONF"; then
+dnl        AC_DEFINE(MOZ_ENABLE_GCONF)
+dnl    fi
 
-    if test "$MOZ_ENABLE_GCONF"
-    then
-        PKG_CHECK_MODULES(MOZ_GCONF, gconf-2.0 >= $GCONF_VERSION gobject-2.0 ,[
-            MOZ_GCONF_LIBS=`$PKG_CONFIG --libs gobject-2.0`
-            MOZ_ENABLE_GCONF=1
-        ],[
-            if test -n "$MOZ_ENABLE_GCONF";
-            then
-                AC_MSG_ERROR([* * * Could not find gconf-2.0 ])
-            else
-                AC_MSG_WARN([Many automated tests will fail with --disable-gconf. See bug 1167201.])
-            fi
-        ])
-    fi
-
-    if test "$MOZ_ENABLE_GCONF"; then
-        AC_DEFINE(MOZ_ENABLE_GCONF)
-    fi
-
-    AC_SUBST(MOZ_ENABLE_GCONF)
-fi
+dnl    AC_SUBST(MOZ_ENABLE_GCONF)
+dnl fi
 
 dnl ========================================================
 dnl = libproxy support
@@ -2366,9 +2366,9 @@
     MOZ_SRTP=1
     AC_DEFINE(MOZ_SCTP)
     AC_DEFINE(MOZ_SRTP)
-    if test -n "$MOZ_X11"; then
-      MOZ_WEBRTC_X11_LIBS="-lXext -lXdamage -lXfixes -lXcomposite"
-    fi
+dnl    if test -n "$MOZ_X11"; then
+dnl      MOZ_WEBRTC_X11_LIBS="-lXext -lXdamage -lXfixes -lXcomposite"
+dnl    fi
 fi
 
 dnl ========================================================
@@ -2386,7 +2386,7 @@
 AC_SUBST(MOZ_WEBRTC_HARDWARE_AEC_NS)
 AC_SUBST(MOZ_SCTP)
 AC_SUBST(MOZ_SRTP)
-AC_SUBST_LIST(MOZ_WEBRTC_X11_LIBS)
+dnl AC_SUBST_LIST(MOZ_WEBRTC_X11_LIBS)
 
 dnl Use integers over floats for audio on B2G and Android
 dnl (regarless of the CPU architecture, because audio
@@ -4028,11 +4028,11 @@
 
 # Define macros for cairo-features.h
 TEE_SURFACE_FEATURE="#define CAIRO_HAS_TEE_SURFACE 1"
-if test "$MOZ_X11"; then
-    XLIB_SURFACE_FEATURE="#define CAIRO_HAS_XLIB_SURFACE 1"
-    XLIB_XRENDER_SURFACE_FEATURE="#define CAIRO_HAS_XLIB_XRENDER_SURFACE 1"
-    PS_SURFACE_FEATURE="#define CAIRO_HAS_PS_SURFACE 1"
-fi
+dnl if test "$MOZ_X11"; then
+dnl    XLIB_SURFACE_FEATURE="#define CAIRO_HAS_XLIB_SURFACE 1"
+dnl    XLIB_XRENDER_SURFACE_FEATURE="#define CAIRO_HAS_XLIB_XRENDER_SURFACE 1"
+dnl    PS_SURFACE_FEATURE="#define CAIRO_HAS_PS_SURFACE 1"
+dnl fi
 if test "$_HAVE_FREETYPE2"; then
     FT_FONT_FEATURE="#define CAIRO_HAS_FT_FONT 1"
     MOZ_ENABLE_CAIRO_FT=1
@@ -4087,9 +4087,9 @@
 AC_SUBST(QT_SURFACE_FEATURE)
 AC_SUBST(TEE_SURFACE_FEATURE)
 
-if test "$MOZ_X11"; then
-    MOZ_CAIRO_OSLIBS="$MOZ_CAIRO_OSLIBS $XLDFLAGS -lXrender"
-fi
+dnl if test "$MOZ_X11"; then
+dnl    MOZ_CAIRO_OSLIBS="$MOZ_CAIRO_OSLIBS $XLDFLAGS -lXrender"
+dnl  fi
 
 CAIRO_FEATURES_H=gfx/cairo/cairo/src/cairo-features.h
 
@@ -4601,34 +4601,34 @@
 AC_CHECK_FUNCS(posix_fadvise posix_fallocate)
 
 dnl Check for missing components
-if test "$MOZ_X11"; then
+dnl if test "$MOZ_X11"; then
     dnl ====================================================
     dnl = Check if X headers exist
     dnl ====================================================
-    _SAVE_CFLAGS=$CFLAGS
-    CFLAGS="$CFLAGS $XCFLAGS"
-    AC_TRY_COMPILE([
-        #include <stdio.h>
-        #include <stdlib.h>
-        #include <X11/Xlib.h>
-        #include <X11/Intrinsic.h>
-        #include <X11/extensions/XShm.h>
-    ],
-    [
-        Display *dpy = 0;
-        if ((dpy = XOpenDisplay(NULL)) == NULL) {
-            fprintf(stderr, ": can't open %s\n", XDisplayName(NULL));
-            exit(1);
-        }
-    ], [],
-    [ AC_MSG_ERROR([Can't find X headers (install libxt-dev (Debian/Ubuntu), libXt-devel (Fedora), or xorg-x11-libXt-devel (SuSE)).]) ])
-    CFLAGS="$_SAVE_CFLAGS"
-
-    if test -n "$MISSING_X"; then
-        AC_MSG_ERROR([ Could not find the following X libraries: $MISSING_X ]);
-    fi
+dnl    _SAVE_CFLAGS=$CFLAGS
+dnl    CFLAGS="$CFLAGS $XCFLAGS"
+dnl    AC_TRY_COMPILE([
+dnl        #include <stdio.h>
+dnl        #include <stdlib.h>
+dnl        #include <X11/Xlib.h>
+dnl        #include <X11/Intrinsic.h>
+dnl        #include <X11/extensions/XShm.h>
+dnl    ],
+dnl    [
+dnl         Display *dpy = 0;
+dnl         if ((dpy = XOpenDisplay(NULL)) == NULL) {
+dnl             fprintf(stderr, ": can't open %s\n", XDisplayName(NULL));
+dnl             exit(1);
+dnl         }
+dnl     ], [],
+dnl     [ AC_MSG_ERROR([Can't find X headers (install libxt-dev (Debian/Ubuntu), libXt-devel (Fedora), or xorg-x11-libXt-devel (SuSE)).]) ])
+dnl     CFLAGS="$_SAVE_CFLAGS"
+
+dnl     if test -n "$MISSING_X"; then
+dnl         AC_MSG_ERROR([ Could not find the following X libraries: $MISSING_X ]);
+dnl     fi
 
-fi # MOZ_X11
+dnl fi # MOZ_X11
 
 fi # COMPILE_ENVIRONMENT
 
